frontend/app-development/features/administration/components/Administration.module.css

```diff
@@ -1,35 +1,77 @@
-.avatar {
-  max-height: '2em';
+.administration {
+  background-color: #e6eff8;
+  position: relative;
 }
-.sidebarHeader {
-  margin-bottom: 20;
-  font-size: 20;
-  font-weight: 500;
+
+.administration::before {
+  /* webpackIgnore: true */
```

Comment:
I used webpackIgnore as a temporary fix since our current webpack configuration doesn't seem to support relative paths. I'll check if I can fix the webpack configuration